.gitignore

```diff
@@ -28,6 +28,7 @@ examples/*/tb_logs/
 examples/*/logs/
 examples/*/demo*/
 examples/*/configs_experiment/
+results/*
```

Comment:
This should not be the dir for results/outputs. Outputs/results should be in `examples/*/outputs/`.